packages/knip/test/plugins/cypress-multi-reporter.test.ts

```diff
@@ -0,0 +1,28 @@
+import { test } from 'bun:test';
+import assert from 'node:assert/strict';
+import { main } from '../../src/index.js';
+import { resolve } from '../../src/util/path.js';
+import baseArguments from '../helpers/baseArguments.js';
+import baseCounters from '../helpers/baseCounters.js';
+
+const cwd = resolve('fixtures/plugins/cypress-multi-reporter');
+
+test('Find dependencies with the cypress-multi-reporter plugin', async () => {
+  const { issues, counters } = await main({
+    ...baseArguments,
+    cwd,
+  });
+
+  assert(issues.unlisted['cypress.config.ts']['@nrwl/cypress/plugins/cypress-preset']);
+  assert(issues.unlisted['cypress/support/commands.ts']['@faker-js/faker']);
+  assert(issues.unlisted['cypress/support/e2e.ts']['@testing-library/cypress/add-commands']);
+  assert(issues.unlisted['cypress.config.ts']['@testing-library/my-fake-reporter']);
```

Comment:
Is there anyway I can make this report as 
```ts
assert(issues.unlisted['reporter-config.json']['@testing-library/my-fake-reporter']);
```
instead?

Reply:
Unfortunately this isn't supported (yet).